docker-image/v1.3/debian-elasticsearch/conf/kubernetes.conf

```diff
@@ -188,5 +188,8 @@
 <filter kubernetes.**>
   @type kubernetes_metadata
   @id filter_kube_metadata
+  kubernetes_url "#{ENV['FLUENT_FILTER_KUBERNETS_URL'] || https://#{ENV['KUBERNETES_SERVICE_HOST']}:#{ENV['KUBERNETES_SERVICE_PORT']}/api}"
```

Comment:
https://github.com/fluent/fluentd-kubernetes-daemonset/pull/279/files/532befe36d84250a8d50b8cc8aeeaccd61206b1d#diff-bf350a335c264690ed81e269cca50f23

All others are the same.

Reply:
Sorry for the late reply.
@ganmacs I don't understand your comment. Could you be more explicit in what change you are looking for please?

Reply:
Never mind, I finally understood! Sorry.